src/bootchooser.c

```diff
@@ -508,9 +501,13 @@ static gboolean uboot_set_primary(RaucSlot *slot) {
 	/* Add updated slot as first entry in new boot order */
 	g_string_append(order_new, slot->bootname);
 
-	res = uboot_env_get("BOOT_ORDER", &order_current, NULL);
-	if (!res && !bootchooser_order_primay(slot, &order_current))
-		goto out;
+	res = uboot_env_get("BOOT_ORDER", &order_current, &ierror);
+	if (!res) {
+		g_message("Unable to obtain BOOT_ORDER");
```

Comment:
Maybe indicate that this is not cirtical and we fall back to a default order?

Reply:
I assume changing it to 
``` c
g_message("Unable to obtain BOOT_ORDER, using defaults");
```

would be sufficient, as the log level indicates being non-critical?